### ISSUE TITLE
WSL terminal not displaying anything after installation of Ubuntu on Windows 10

### ISSUE THREAD

**Issue by ace411** [Microsoft/vscode#48331]
Hello. I recently installed Ubuntu for windows after configuring WSL on my machine. I changed my vscode settings to enable me use the new bash terminal but all I got was a blank screen. It works fine outside vscode though.
![terminal_issues](https://user-images.githubusercontent.com/11040337/39089944-9624b9ae-45da-11e8-82fb-1e888b4608f1.JPG)
![terminal_issues_2](https://user-images.githubusercontent.com/11040337/39089945-965d9f08-45da-11e8-92c1-4daaadb4f1eb.JPG)

- VSCode Version:  1.22
- OS Version: Windows 10 Professional

Steps to Reproduce:

1. Restart VSCode multiple times
2. Change the terminal settings

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by DarthSpock** [comment]
A workaround could be to use the Linux version of VSCode in your Ubuntu instance provided you are running build 17134+ (the April update). The terminal works great there.  I also noticed you used bash.exe. Try using wsl.exe to see if that makes a difference. If Ubuntu is the first or only distro installed, it will call Ubuntu.

**Comment by Tyriar** [comment]
Yeah I'd say try wsl.exe instead. We should be recommending that instead of bash.exe on newer versions of Windows.

**Comment by Nealsvanderven** [comment]
I ran into the same issues as well. I managed to 'fix' it by migrating from the x86 to x64 version. Both Bash.exe and wsl.exe are working fine now.
